### DISCUSSION TITLE
Weekly Meeting Oct. 8, 2023

### DISCUSSION THREAD

**Discussion by kikuchiharuma** [project-theory-probe/ptp-voyage#6] [General]
Meet: Project Theory Probe Weekly Meeting
Wednesday, November 8 · 14:00 – 15:00
Time zone: Asia/Tokyo
Google Meet joining info
Video call link: https://meet.google.com/qyd-seed-aew
Or dial: ‪(JP) +81 3-4545-0450‬ PIN: ‪100 286 257 6031‬#
More phone numbers: https://tel.meet/qyd-seed-aew?pin=1002862576031

**Comment by shotaro8yagi** [comment]
11/8メモ
＜コラム案＞

- H2ロケット←→イーロンマスク
  - https://www.youtube.com/watch?v=4UgHtFKALZU
- 探査機ルーシー
  - https://www.space.com/nasa-lucy-asteroid-dinky-3-space-rocks-contact-binary
- ボイジャー号
  - もっと早く終わるはずだった
  - ヘリオポーズを越えたのは副産物
- 19世紀ライト兄弟
  - https://www.youtube.com/watch?v=oVUjh0O7YEc
  - 未知のものにゴール設定する虚しさ
- インターネットの歴史
  - https://en.wikipedia.org/wiki/Gopher_(protocol)
- 深海
  - https://www.youtube.com/watch?v=8QH5hBOoz08
- 山の上
  - https://www.youtube.com/watch?v=8QH5hBOoz08

**Comment by kikuchiharuma** [comment]
連載の次回のネタ
- 今月の一冊: プラグマティズム関連からセレクト
- 伝道師: エリック・レイモンド (テレホンショッキング方式)
